internal/pkg/plugin/shell_plugin_installer.go

```diff
@@ -0,0 +1,19 @@
+package plugin
+
+import (
+	"github.com/hashicorp/go-version"
+)
+
+type ShellPluginInstaller struct {
+	Name          string
+	RepositoryDir string
+	InstallDir    string
+}
+
+func (s *ShellPluginInstaller) CheckVersion(_ *version.Version) error {
+	return nil
```

Comment:
Previously this was because I didn't want to enforce a specific shell script. But since we're going with just bash scripts now, I'll add a version check for bash.